### ISSUE TITLE
Confusing message on `application-duality` rule

### ISSUE THREAD

**Issue by maxonfjvipon** [objectionary/lints#409]
I have this XMIR:
```xml
<o base="Q.org.eolang.tuple">
    <o base="Q.org.eolang.tuple.empty"/>
    <o as="α1" base="Q.org.eolang.number" line="10" pos="8">
       <o base="Q.org.eolang.bytes">3F-F0-00-00-00-00-00-00</o>
    </o>
    <o base="Q.org.eolang.number">
       <o base="Q.org.eolang.bytes">3F-F0-00-00-00-00-00-00</o>
    </o>
 </o>
```
When it's linter I'm getting this: `The anonymous object cannot have both @name and @as attributes as this is prohibited due to duality in the application, please use only one (application-duality)`

The message is confusing since:
1. there's no anonymous object
2. `@as` and `@name` existence are not related to application duality

**Comment by h1alexbel** [comment]
@maxonfjvipon object anonymousless depends on the presence of `@name` attribute. `Q.org.eolang.tuple` object has only `@base`. About `@as` and `@name`, I see that `Q.org.eolang.tuple` object has three children, one of them has `@as` attribute, while two others don't. Am I missing something?

**Comment by maxonfjvipon** [comment]
> I see that Q.org.eolang.tuple object has three children, one of them has @as attribute, while two others don't.

You're right, `application-duality` must be caught here, what does `@name` have to do with it? 
The message `object cannot have both @name and @as attributes` is wrong. The `application-duality` must be caught if some arguments have `@as`, some don't. There's nothing about `@name`

**Comment by h1alexbel** [comment]
@maxonfjvipon yes, should we delete it from the message?

**Comment by maxonfjvipon** [comment]
@h1alexbel I think the message should explain what is wrong, what application duality means, so removing `@name` part is not enough

**Comment by h1alexbel** [comment]
@maxonfjvipon looks like we added it by mistake (it should be `@base` part instead)

**Comment by 0crat** [comment]
@maxonfjvipon Thank you for submitting this ticket. Unfortunately, it was not accepted by the team. As per our policy, this results in a deduction of 16 points. Your current balance is now [+37](https://www.eolang.org/zerocracy/objectionary-vitals.html). While this outcome is disappointing, please don't be discouraged. Your efforts in reporting issues are valuable for improving our project's quality. We encourage you to continue [reporting](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) potential bugs or enhancements, but please be more careful in your submissions. Remember, [bugs are welcome](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) when they help us improve our product.
